tests/unittests/CoreGraphics.drawing/CTDrawingTests.cpp

```diff
@@ -237,26 +237,23 @@ INSTANTIATE_TEST_CASE_P(TestAlignmentLineBreakMode,
                                            ::testing::ValuesIn(c_writingDirections),
                                            ::testing::ValuesIn(c_fontSizes)));
 
-class RotateCTM : public WhiteBackgroundTest, public ::testing::WithParamInterface<CGFloat> {
-    CFStringRef CreateOutputFilename() {
-        CGFloat rotation = GetParam();
-        return CFStringCreateWithFormat(nullptr, nullptr, CFSTR("TestImage.RotateCTM.%.02f.png"), rotation);
-    }
-};
+class Transform : public WhiteBackgroundTest,
+                  public ::testing::WithParamInterface<::testing::tuple<CGAffineTransform, CGAffineTransform>> {};
 
-DISABLED_DRAW_TEST_P(RotateCTM, TestRotatingCTM) {
+DISABLED_DRAW_TEST_P(Transform, TestMatrices) {
     CGContextRef context = GetDrawingContext();
     CGRect bounds = GetDrawingBounds();
 
-    CGContextRotateCTM(context, GetParam());
+    CGContextSetTextMatrix(context, ::testing::get<0>(GetParam()));
+    CGContextConcatCTM(context, ::testing::get<1>(GetParam()));
 
     // Creates path with current rectangle
     woc::unique_cf<CGMutablePathRef> path{ CGPathCreateMutable() };
     CGPathAddRect(path.get(), nullptr, bounds);
 
     // Create style setting to match given alignment
     CTParagraphStyleSetting setting[1];
-    CTTextAlignment alignment = kCTLeftTextAlignment;
+    CTTextAlignment alignment = kCTCenterTextAlignment;
```

Comment:
I thought it made it easier to see what was going wrong with drawing.  Gonna leave in case someone (probably me) breaks affine transforms in the future